src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -0,0 +1,446 @@
+| ScaleFactor                          | RECOMMENDED           | [array][] of [numbers][]               | Scale factor for each frame.                                                                                    |
+| ScatterFraction                      | RECOMMENDED           | [array][] of [numbers][]               | Scatter fraction for each frame.                                                                                |
+| DecayCorrectionFactor                | RECOMMENDED           | [array][] of [numbers][]               | Decay correction factor for each frame.                                                                         |
+| PromptRate                           | RECOMMENDED           | [array][] of [numbers][]               | Prompt rate for each frame.                                                                                     |
+| RandomRate                           | RECOMMENDED           | [array][] of [numbers][]               | Random rate for each frame.                                                                                     |
+| SinglesRate                          | RECOMMENDED           | [array][] of [numbers][]               | Singles rate for each frame.                                                                                    |
+
+#### Example (`*_pet.json`)
+
+```JSON
+{
+	"Manufacturer": "Siemens",
+	"ManufacturersModelName": "High-Resolution Research Tomograph (HRRT, CTI/Siemens)",
+	"BodyPart": "Brain",
+	"BodyWeight": 21,
+	"BodyWeightUnit": "kg",
+	"Unit": "Bq/mL",
+	"TracerName": "CIMBI-36",
+	"TracerRadionuclide": "C11",
+	"TracerMolecularWeight": 380.28,
+	"TracerMolecularWeightUnit": "g/mol",
+	"TracerInjectionType": "bolus",
+	"InjectedRadioactivity": 573,
+	"InjectedRadioActivityUnit": "MBq",
+	"InjectedMass": 0.62,
+	"InjectedMassUnit": "ug",
+	"SpecificRadioactivity": 929.6,
+	"SpecificRadioactivityUnit": "MBq/ug",
+	"ModeOfAdministration": "bolus",
+	"MolarActivity": 353.51,
+	"MolarActivityUnit": "GBq/umol",
+	"MolarActivityMeasTime": "13:04:42",
+	"TimeZero": "13:04:42",
+	"ScanStart": 0,
+	"InjectionStart": 0,
+	"FrameTimesStart": [0, 10, 20, 30, 40, 50, 60, 80, 100, 120, 140, 160, 180, 240, 300, 360, 420, 480, 540, 660, 780, 900, 1020, 1140, 1260, 1380, 1500, 1800, 2100, 2400, 2700, 3000, 3300, 3600, 3900, 4200, 4500, 4800, 5100, 5400, 5700, 6000, 6300, 6600, 6900],
+	"FrameDuration": [10, 10, 10, 10, 10, 10, 20, 20, 20, 20, 20, 20, 60, 60, 60, 60, 60, 60, 120, 120, 120, 120, 120, 120, 120, 120, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300],
+	"AcquisitionMode": "list mode",
+	"ImageDecayCorrected": true,
+	"ImageDecayCorrectionTime": 0,
+	"ReconMethodName": "3D-OSEM-PSF",
+	"ReconMethodParameterLabels": ["subsets","iterations"],
+	"ReconMethodParameterUnit": ["none","none"],
+	"ReconMethodParameterValues": [16,10],
+	"ReconFilterType": "none",
```

Comment:
Adding ReconFilterSize according to the specification. Should we consider to make this conditional in the validator, so if "ReconFilterType" = "none", then "ReconFilterSize" is not required? 

```suggestion
	"ReconFilterType": "none",
	"ReconFilterSize": 0,
```

Reply:
Hi @mnoergaard I agree that _ReconFilterSize_ should be conditional to the value of _ReconFilterType_. An alternative would be to explicitly say in the text that it MUST be 0 if _ReconFilterType_ is "none". I guess that whichever of the two works better in the validator could be used